Xamarin.PropertyEditing.Mac/Controls/Custom/UnfocusableButton.cs

```diff
@@ -33,25 +33,31 @@ protected override void Dispose (bool disposing)
 
 		public override void MouseDown (NSEvent theEvent)
 		{
-			switch (theEvent.Type) {
+			if (Enabled) {
```

Comment:
I'd recommend inverting stuff like this so instead of indenting everything, we simply have a
```csharp
if (!Enabled)
    return;
```
at the top